src/test/java/io/nats/client/ConnectTests.java

```diff
@@ -396,26 +355,17 @@ public void testAsyncConnectionWithReconnect() throws IOException, InterruptedEx
         try {
             Nats.connectAsynchronously(options, true);
 
-            // No server at this point, let it fail and try to start over
-            try {
-                Thread.sleep(1000);
-            } catch (Exception exp) {
-
-            }
+            sleep(5000); // No server at this point, let it fail and try to start over

```

Comment:
sleep macro just cleaner.

Reply:
could be a bit more deterministic and use `options.getConnectionTimeout() + 100` in case the default timeout changes.